src/components/ProfileCard/ProfileCard.constants.ts

```diff
@@ -0,0 +1,97 @@
+import EmojiEventsIcon from '@mui/icons-material/EmojiEvents';
+import PersonIcon from '@mui/icons-material/Person';
+import SchoolIcon from '@mui/icons-material/School';
+import StarIcon from '@mui/icons-material/Star';
+
+import type { ProfileCardSize, ProfileCardVariant, ProfileStats } from './ProfileCard.types';
+
+export const PROFILE_CARD_SIZES: Record<ProfileCardSize, ProfileCardSize> = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+export const PROFILE_CARD_VARIANTS: Record<ProfileCardVariant, ProfileCardVariant> = {
+  default: 'default',
+  compact: 'compact',
+  expanded: 'expanded',
+} as const;
+
+export const SIZE_CONFIGS = {
+  small: {
+    avatarSize: 48,
```

Comment:
SIZE_CONFIGS uses concrete numeric values for avatarSize (48). These numeric values will be used as raw px values in the styles file and violate the project's rule of no hardcoded px and using the MUI theme spacing / tokens instead.

**Sugerencia:**
```typescript
Store sizes in theme-relative units (spacing steps or `rem`) instead of raw px. Example change in constants:

// use theme spacing units (numbers) instead of px
small: { avatarSpacing: 6, paddingUnits: 2, ... }

Then in styles use `width: theme.spacing(sizeConfig.avatarSpacing)` and `padding: theme.spacing(sizeConfig.paddingUnits)` so all sizes come from the theme.
```